### PR TITLE
Docs: Muting checks

### PR DESCRIPTION
**Pull request by dwaynebailey**
Add screenshots and icons to guide people on how to mute checks and where checks appear.

### CODE REVIEW

**Comment by phlax** [comment]
lgtm

File: docs/features/checks.rst [hunk 1]
```diff
@@ -35,9 +42,10 @@ Overriding Quality Checks
 -------------------------
 
 It is possible to override the quality check if the translation is correct.
```
Comment (unho) [inline docs/features/checks.rst]: I would suggest to replace all appearances of "override" and others by **"mute"**.

File: docs/features/checks.rst [hunk 2]
```diff
@@ -35,9 +42,10 @@ Overriding Quality Checks
...
-Reviewers are able to remove the check for a certain string to indicate that
-the string is correctly translated. This avoids having to recheck the same
-checks multiple times.
+Reviewers are able to remove the check for a certain string, by clicking on the
+|icon:icon-block| button, to indicate that the string is correctly translated.
+This avoids having to recheck the same checks multiple times.  You can off
```
Comment (unho) [inline docs/features/checks.rst]: check again and again the same checks.

File: docs/features/checks.rst [hunk 3]
```diff
@@ -35,9 +42,10 @@ Overriding Quality Checks
...
+course unmute the test using the |icon:icon-block-muted| button.
```
Comment (unho) [inline docs/features/checks.rst]: unmute any muted check

File: docs/features/checks.rst [hunk 4]
```diff
@@ -13,15 +13,22 @@ Pootle administrators should indicate the correct project type (GNOME, KDE,
...
 checks will be immediately reported, preventing you from automatically
-continuing until the issues have been resolved or disregarded as false
-positives.
+continuing until the issues have been resolved or muted as false positives by
+using the mute |icon:icon-block| button. Non-crtical errors flag potential
+problems but can be ignored or muted as needed.
+
+.. image:: ../_static/failing-checks.png
 
 To understand the meaning of each test, Pootle displays the failing tests
```
Comment (unho) [inline docs/features/checks.rst]: failing checks

File: docs/features/checks.rst [hunk 5]
```diff
@@ -35,9 +42,10 @@ Overriding Quality Checks
...
-the string is correctly translated. This avoids having to recheck the same
-checks multiple times.
+Reviewers are able to remove the check for a certain string, by clicking on the
+|icon:icon-block| button, to indicate that the string is correctly translated.
+This avoids having to recheck the same checks multiple times.  You can off
+course unmute the test using the |icon:icon-block-muted| button.
 
-If the translation is changed, this information is discarded to ensure that the
-new translation is tested again for any possible issues.
+If the translation is changed, the override information is discarded to ensure
```
Comment (unho) [inline docs/features/checks.rst]: mute information

File: docs/features/checks.rst [hunk 6]
```diff
@@ -35,9 +42,10 @@ Overriding Quality Checks
...
+that the new translation is tested again for any possible issues.
```
Comment (unho) [inline docs/features/checks.rst]: checked again

File: docs/features/checks.rst [hunk 7]
```diff
@@ -13,15 +13,22 @@ Pootle administrators should indicate the correct project type (GNOME, KDE,
...
-extra failing checks can be accessed by clicking the ``+`` button located
-right below the navigation breadcrumbs. Clicking on the name of a test
-will step you through the translations that fail the test.
+Critical checks are prominently displayed through the browsing UI.
+
+.. image:: ../_static/critical_errors.png
+
+Any extra failing checks can be accessed by clicking the
+|icon:icon-expand-stats| button located below the navigation breadcrumbs.
+Clicking on the name of a test will step you through the translations that fail
```
Comment (unho) [inline docs/features/checks.rst]: of a check

File: docs/features/checks.rst [hunk 8]
```diff
@@ -13,15 +13,22 @@ Pootle administrators should indicate the correct project type (GNOME, KDE,
...
+that test.
```
Comment (unho) [inline docs/features/checks.rst]: that check

File: docs/features/checks.rst [hunk 9]
```diff
@@ -13,15 +13,22 @@ Pootle administrators should indicate the correct project type (GNOME, KDE,
...
 
 While in the translation editor, submissions resulting in critical failing
 checks will be immediately reported, preventing you from automatically
-continuing until the issues have been resolved or disregarded as false
-positives.
+continuing until the issues have been resolved or muted as false positives by
+using the mute |icon:icon-block| button. Non-crtical errors flag potential
```
Comment (unho) [inline docs/features/checks.rst]: critical

**Comment by unho** [comment]
gtm once nits and **error** are addressed.

**Comment by unho** [comment]
Also I would use a `docs/_static/failing-checks.png` that displays the failing check in the context of the editor. Current screenshot doesn't clearly state that.

**Pull request merged**